src/controllers/user.js

```diff
@@ -1,20 +1,43 @@
+const UserModel = require('../models/user');
+
 module.exports = {
-  getAllUsers: (req, res) => {
-    res.send('All users');
+  getAllUsers: async (req, res) => {
+    try {
+      const users = await UserModel.find();
+      res.json(users);
+    } catch (err) {
+      res.status(422).json({ message: err.message ?? err });
+    }
```

Comment:
Can you add a comment above this function and `updateUser`?
It can say: `// Temporary code added, to be merged and updated in upcoming PR`